### ISSUE TITLE
Unexpected warning on ControlStructureSpacing with T_CASE token

### ISSUE THREAD

**Issue by CZechBoY** [slevomat/coding-standard#620]
With this configuration:
```xml
    <rule ref="SlevomatCodingStandard.ControlStructures.ControlStructureSpacing">
        <properties>
            <property name="linesCountAroundControlStructure" type="int" value="1"/>
            <property name="linesCountAfterLastControlStructure" type="int" value="1"/>
            <property name="linesCountBeforeFirstControlStructure" type="int" value="0"/>
            <property name="tokensToCheck" type="array">
                <element value="T_CASE"/>
            </property>
        </properties>
    </rule>
```

and this php:
```php
switch ($abcd)
{
    case 'x':
        $statements[$node->id] = new Start(...);
        break;

    case 'a': //on this line is:   Expected 1 lines after "case", found 0.
        try {
            $a = $this->getXyz($node);
        } catch (\InvalidArgumentException $e) {
            $a = null;
        }
        $statements[$node->id] = new A($a);
        break;

    case 'b':
       ...

    default:
       throw new \InvalidArgumentException('...');
}
```

Not expected warning is mentioned in php code snippet.

**Comment by kukulich** [comment]
The sniff was not intented to check/fix `case` or `default` tokens. It should probably throw error when you use unsupported tokens...
